packages/osc-ui/src/components/Modal/Modal.stories.tsx

```diff
@@ -5,7 +5,56 @@ import { Modal } from './Modal';
 
 export default {
     title: 'osc-ui/Modal',
-    component: Modal
+    component: Modal,
+    parameters: {
+        docs: {
+            description: {
+                component:
+                    'A window overlaid on either the primary window or another dialog window, rendering the content underneath inert.'
+            }
+        }
+    },
+    argTypes: {
+        children: {
+            table: {
+                disable: true
+            }
+        },
+        disableOutsideClick: {
+            description: 'Sets whether the user can click outside of the modal to close it'
+        },
+        hideFooterCloseButton: {
+            description: 'Sets whether or not to hide the close button in the footer'
+        },
```

Comment:
```suggestion
        hideFooterCloseButton: {
            description: 'Sets whether or not to prevent rendering of the close button in the footer'
        },
```

Would `render` be a better term than `hide`? hide to me implies the button is there but perhaps with `display: none;` and could still be clicked programatically?

Reply:
Yeah that's a good point! I'll update it